content/applications/sales/crm/acquire_leads/send_quotes.rst

```diff
@@ -1,34 +1,149 @@
+
+When the quotation is ready to deliver to the customer, click :guilabel:`Send by Email`. This opens
+a pop-up window with a pre-configured email message. Information from the quotation, including the
+contact information, total cost, and quotation title will be imported from the quotation. A PDF of
+the quotation is added as an attachment to the email.
+
+.. note::
+   A pre-loaded template is used to create the email message. To alter the template, click the
+   internal link to the right of the :guilabel:`Load template` field. To select a new template,
+   select an option from the :guilabel:`Load template` drop-down.
+
+Make any necessary changes to the email, then click :guilabel:`Send`. A copy of the message is added
+to the *Chatter* of the of the record.
+
+.. seealso::
+   - `Quotation templates </applications/sales/sales/send_quotations/quote_template/>`_
+   - `Optional products </applications/sales/sales/send_quotations/optional_products/>`_
+   - `Quotation deadlines </applications/sales/sales/send_quotations/deadline/>`_
+
+Mark an opportunity won or lost
+===============================
+
+In order to keep the pipeline up to date and accurate, opportunities need to be identified as won
+or lost once a customer has responded to a quotation.
+
+From the quotation form, return to the opportunity using the breadcrumbs at the top left of the
+record. Or navigate to :menuselection:`CRM app --> Sales --> My Pipeline` and click on the correct
+opportunity to open it.
+
+At the top left of the form, click on either :guilabel:`Won` or :guilabel:`Lost`.
+
+If the opportunity it marked *Won*, a green *Won* banner is added to the record, and it is moved to
+the :guilabel:`Won` stage.
 
-.. image:: send_quotes/send_quotes02.png
-    :align: center
+Marking an opportunity as *Lost*, opens the :guilabel:`Lost Reason`. From the drop-down, choose an
+existing lost reason. If no applicable reason is available, create a new one by entering it into the
+:guilabel:`Lost Reason` field, and clicking :guilabel:`Create`. Additional notes and comments can be
+added below the lost reason designated in the :guilabel:`Lost Reason` field.
 
-You will find all your quotes to that specific opportunity under the
-*Quotations* menu on that page.
+When all the desired information has been entered in the Lost Reason pop-up window, click Submit.
 
-.. image:: send_quotes/send_quotes03.png
-    :align: center
+Upon clicking :guilabel:`Submit`, the pop-up window disappears, and Odoo returns to the lead detail
+form, where a new red :guilabel:`Lost` banner is now present in the upper-right corner of the lead.
 
-Mark them won/lost
-==================
+Once an opportunity is marked as *lost*, it is no longer considered active and is removed from the
+pipeline. In order to view a *lost* lead from the pipeline, select either :guilabel:`Lost` or
+:guilabel:`Archived` from the :guilabel:`Search bar`.`
```

Comment:
```suggestion
Once an opportunity is marked as *lost*, it is no longer considered active, and it is removed from
the pipeline.

In order to view a *lost* lead/opportunity from the pipeline, click the :guilabel:`down arrow icon`
to the right of the search bar, and select either :guilabel:`Lost` or :guilabel:`Archived` from the
drop-down menu that appears.
```

Reply:
Need to change this up slightly, as you can't view a lost lead from the pipeline, only an opportunity. But accepting the rest of the changes